pkg/cmd/create/create.go

```diff
@@ -751,6 +783,26 @@ func (o *Options) watchNamespaceStart() error {
 	return nil
 }
 
+func (o *Options) IfPodIsFailedShareLogs(pod *corev1.Pod, previewNamespace string) error {
+	if pod.Status.Phase == corev1.PodFailed {
+		log.Logger().Infof("found pod %s in namespace %s in state %s", pod.Name, previewNamespace, pod.Status.Phase)
+
+		logs := o.KubeClient.CoreV1().Pods(previewNamespace).GetLogs(pod.Name, &corev1.PodLogOptions{})
+		stream, err := logs.Stream(context.Background())
+		if err != nil {
+			return err
+		}
+		defer stream.Close()
+
+		scanner := bufio.NewScanner(stream)
+		for scanner.Scan() {
+			m := scanner.Text()
+			return fmt.Errorf("failed pod %s in namespace %s: %s", pod.Name, previewNamespace, m)
```

Comment:
Shouldn't this line be below the for loop?

Reply:
Moved outside of the loop